src/store/CategoricalDisplayStore.js

```diff
@@ -285,5 +286,6 @@ export default class CategoricalDisplayStore {
         ResultsPanel: this.ResultsPanel,
         FilterPanel: this.FilterPanel,
         SearchInput: this.SearchInput,
+        isMobile: this.mainStore.chart.isMobile,
```

Comment:
maybe it's better to get chart as a `getter` like `get chart(){return this.mainStore.chart}`